spec/features/authentication/edit_password_spec.rb

```diff
@@ -41,4 +41,25 @@
 
     expect(page).to have_content "Sign out"
   end
+
+  it "stays on the same page and does not change the password, if the confirmation does not match" do
+    visit edit_user_registration_path
+
+    fill_in "Current password", with: user.password
+    fill_in "Password", with: pw
+    fill_in "Password confirmation", with: "different password"
+
+    click_on "Submit"
+
+    expect(page).to have_content "Password confirmation doesn't match Password"
```

Comment:
do we expect this capitalisation?

Reply:
I don't like it but it's what Rails gave me; let me know if you know a way to override it.